Add doc for precompiled demos (#5596)
CGAL/cgal: Documentation/doc/Documentation/windows.txt

folder of the package(s) that you are using to learn how to specify \cgal and additional third party
dependencies.

\section sec-win-demo Trying CGAL algorithms using a Precompiled Demo

> \cgal provides a few demos providing a graphical user interfaces for some \cgal algorithms.
> All precompiled demos are available for download in the \ref packages page.

Reviewer: ```suggestion
\cgal provides a few demos offering a graphical user interface for some \cgal algorithms.
All precompiled demos are available for download on the \ref packages page.
```
